src/message.rs

```diff
@@ -1215,7 +1215,7 @@ pub fn set_msg_failed(context: &Context, msg_id: MsgId, error: Option<impl AsRef
         }
         if let Some(error) = error {
             msg.param.set(Param::Error, error.as_ref());
-            error!(context, "{}", error.as_ref());
+            error!(context, "Message failed: {}", error.as_ref());
```

Comment:
Maybe don't display the error to the user at all in this case? Failed message with a red cross in the corner is enough.

Reply:
yes, i am anyway a little skeptical of these phased-in ephemeral error pop-ups.  If anything i'd rather like to have an error-chat or so.  If @r10s agrees let's turn it into a warning.

Reply:
yip, i agree.

on iOS, these ephemeral error pop-up-toasts were never displayed at all, this is a long-standing issue, there is no "toast" on ios, however, meanwhile i also think, that it is not really needed, i never missed that, the red-cross beside failed messages are just fine.